client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -484,22 +510,22 @@ export class PlanFeatures2023Grid extends Component {
 					<PlanFeatures2023GridFeatures
 						features={ jpFeatures }
 						planName={ planName }
-						translate={ translate }
+						domainName={ domainName }
 					/>
 				</Container>
 			);
 		} );
 	}
 
-	renderPlanStorageOptions( planPropertiesObj, { isMobile } = {} ) {
+	renderPlanStorageOptions( planPropertiesObj: PlanProperties[], options?: PlanRowOptions ) {
 		const { translate } = this.props;
 		return planPropertiesObj.map( ( properties ) => {
-			if ( isMobile && isWpcomEnterpriseGridPlan( properties.planName ) ) {
+			if ( options?.isMobile && isWpcomEnterpriseGridPlan( properties.planName ) ) {
 				return null;
 			}
 
 			const { planName, storageOptions } = properties;
-			const storageJSX = storageOptions.map( ( storageFeature ) => {
+			const storageJSX = storageOptions.map( ( storageFeature: any ) => {
```

Comment:
We do know how `storageFeature` is used so we can provide a more clear type here than `any`. It's passed to `getStorageStringFromFeature()` which accepts only strings.

```suggestion
			const storageJSX = storageOptions.map( ( storageFeature: string ) => {
```